### PR TITLE
VACMS 17626 - Handles empty nearby Vet Centers (e.g. Guam locations)

### PR DESCRIPTION
**Pull request by eselkin**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

- If no nearby Vet Centers does not send empty request to MapBox. If no nearby Vet Centers adds message to section that no nearby locations were found.
- Prevents call to MapBox which when empty causes error
- Sitewide team

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17626

## Testing done

- When sending empty request, would call MapBox with bad format and receive 400 error and crash
- Check Guam Vet Center locations page (/guam-vet-center/locations) and see that it no longer crashes due to NearbyVetCenter component. Check that Vet Center with nearby locations and see that it performs as it had been (/boston-vet-center/locations)

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

<details>
<summary>Boston Vet Center in PR (same results as dev.va.gov)</summary>
Screenshot 2024-07-24 at 3 27 24 PM
</details>

<details><summary>Guam Vet Center in PR</summary>
Screenshot 2024-07-24 at 3 31 01 PM
</details>

<details><summary>Guam Vet Center in prod (crash)</summary>
Screenshot 2024-07-24 at 3 28 55 PM
</details>

## What areas of the site does it impact?

Vet Center /locations pages

## Acceptance criteria

- [x] NearBy Vet Center Component should load and return no results on Guam
- [ ] Requires accessibility review

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user (unauthenticated)

## Requested Feedback

Check 
error on prod:
/guam-vet-center/locations
and
/memphis-vet-center/locations/
and
as normal on prod
/boston-vet-center/locations/

### CODE REVIEW

**Review by rmessina1010 — APPROVED** [review]
LGTM

File: src/applications/static-pages/facilities/vet-center/NearByVetCenters.jsx [hunk 1]
```diff
@@ -222,15 +225,31 @@ const NearByVetCenters = props => {
...
+      <>
+        <h2
+          className="vads-u-font-size--xl vads-u-margin-top--3 medium-screen:vads-u-margin-top--5 vads-u-margin-bottom--2p5
+                  medium-screen:vads-u-margin-bottom--3"
+          id="other-near-locations-not-found"
+          data-testid="other-near-locations-not-found"
+        >
+          Other nearby Vet Centers
+        </h2>
+        <p>No nearby Vet Center locations found.</p>
```
Comment (randimays) [inline src/applications/static-pages/facilities/vet-center/NearByVetCenters.jsx:239]: Do we need to show this section at all if nearby vet center locations are not found? I'm wondering whether the user would expect to see this section and be confused if it were missing (even if there weren't data for it).

Comment (eselkin) [reply src/applications/static-pages/facilities/vet-center/NearByVetCenters.jsx:239]: We had this discussion around VBA and the decision was that we should show it there. The consensus from Jill and Dave and others was to keep it here as well.

Comment (randimays) [reply src/applications/static-pages/facilities/vet-center/NearByVetCenters.jsx:239]: Gotcha. Thanks!

**Review by randimays — APPROVED** [review]
LGTM

**Pull request merged**